meson.build

```diff
@@ -45,6 +45,24 @@ if all_loaders or get_option('loaders').contains('jpg') == true
     config_h.set10('THORVG_JPG_LOADER_SUPPORT', true)
 endif
 
+if all_loaders or get_option('loaders').contains('webp') == true
+    webp_dep = dependency('libwebp', required: false)
+
+    if get_option('static') == true
+        if all_loaders
+            message('static webp is not available, disable webp loader.')
+        elif get_option('loaders').contains('webp') == true
+            error('static webp is not available, disable webp loader.')
+        endif
+    else
+        if webp_dep.found()
+            config_h.set10('THORVG_WEBP_LOADER_SUPPORT', true)
+        else
```

Comment:
else if all_loaders -> message
else -> error

Now I think we can keep the "all" option in CI script.